### DISCUSSION TITLE
Selecting multiple videos at the same time

### DISCUSSION THREAD

**Discussion by B0XI** [jlesage/docker-handbrake#331] [General]
Hi!
I'm encoding lots of videos, what are located in different folders, and I'd like it to automatically put it in the same folder as the original. So this is why the watch function is not suitable. I can easily do this task on the desktop version, and I hope it is possible in docker, I just do something wrong. Can anyone help me, is it my browser (safari), or is it something else, or it is just not a feature?

**Comment by jlesage** [comment]
The Linux version of HandBrake doesn't have the exact same feature set as the Windows version. This is why this feature is missing.
